Claudinee11/My-Dairy
# adding ui and home page

What does this PR do?
- adds navbar
- add background images
- add footer
- add about us and contact us form

adds more details on properties details
Description of Task to be completed
 - A user should be able to see the home page with different navbar and footer and contact us and about us and also details of my project, they are redirected to the home page.
How should this be manually tested?
- clone the repo and switch to the branch ft-home-ft-home-168192693

enter details in the field and click on the image as a link for more details

- you should be successfully redirected to home.html

What are the relevant pivotal tracker stories?
- 168192693
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/50420731/64016625-5eed9800-cb8c-11e9-88a4-c28f2261a1bc.png)

#### Questions:
